presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -28,6 +28,10 @@
 #include "presto_cpp/main/common/Utils.h"
 #include "presto_cpp/main/connectors/Registration.h"
 #include "presto_cpp/main/connectors/SystemConnector.h"
+#ifdef PRESTO_ENABLE_DATASKETCHES
```

Comment:
Since Iceberg is a default build option in Prestissimo, then I feel these sketch functions should be as well. wdyt ?

@czentgr @PingLiuPing

Reply:
I'm waiting for an input on this one.

Reply:
I feel these can be included by default as Java also has them in presto-main-base
https://github.com/prestodb/presto/blob/master/presto-main-base/src/main/java/com/facebook/presto/operator/scalar/ThetaSketchFunctions.java